pkg/domain/infra/abi/containers_runlabel.go

```diff
@@ -173,6 +173,13 @@ func generateRunlabelCommand(runlabel string, img *libimage.Image, inputName str
 				return ""
 			}
 			return d
+		case "HOME":
+			if h, err := os.UserHomeDir(); err != nil {
+				logrus.Error("Unable to determine user's home directory")
```

Comment:
Suggestion:
```go
    logrus.Warnf("Unable to determine home directory: %q", err)
```
Reasons: (1) this is super unlikely; users might appreciate knowing what happened. (2) since you're proceeding with an empty default, this doesn't really seem like an error